TemplateApplication.xcodeproj/project.pbxproj

```diff
@@ -1260,7 +1226,15 @@
 			repositoryURL = "https://github.com/FelixHerrmann/swift-package-list";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 3.0.10;
+				minimumVersion = 4.1.0;
+			};
+		};
+		56E708332BB06B7100B08F0A /* XCRemoteSwiftPackageReference "SpeziLicense" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziLicense";
+			requirement = {
+				branch = "dependency-list";
+				kind = branch;
```

Comment:
To keep track of open issues: Before we merge this PR, we need to merge SpeziLicense and tag a release there.